### DISCUSSION TITLE
Ubuntu-vm.sh

### DISCUSSION THREAD

**Discussion by eth030** [tteck/Proxmox#2059] [Ideas]
Please can someone creat an Ubuntu-VM install script.
Alongside:
https://github.com/tteck/Proxmox/blob/main/vm/debian-vm.sh
kind regards

**Comment by eth030** [comment]
hope this can get implemented

**Reply by AlexGeek77** [reply]
This project is primarily focused on LXC creations. If you need a standard Ubuntu VM then I would suggest downloading the ISO file and installing it as a VM using the Proxmox GUI. There's an Ubuntu LXC creation script here if you want to use that. 

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/ubuntu.sh)"`

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/2072
